dhash.py

```diff
@@ -80,12 +110,6 @@ def dhash_row_col(image, size=8):
     """Calculate row and column difference hash for given image and return
     hashes as (row_hash, col_hash) where each value is a size*size bit
     integer.
-
-    >>> row, col = dhash_row_col([0,0,1,1,1, 0,1,1,3,4, 0,1,6,6,7, 7,7,7,7,9, 8,7,7,8,9], size=4)
-    >>> format(row, '016b')
-    '0100101111010001'
-    >>> format(col, '016b')
-    '0101001111111001'
```

Comment:
I think we can leave this doctest in as it doesn't depend on external image data, and it's a nice little example of how to use it with a list input.